specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2022-10-01-preview/settings.json

```diff
@@ -0,0 +1,313 @@
+          {
+            "$ref": "common-types.json#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded."
+          },
+          "204": {
+            "description": "OK. The request has succeeded."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common-types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Setting": {
+      "description": "Setting definition.",
+      "discriminator": "kind",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "common-types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "type": "string",
+          "description": "Specifies the kind of settings.",
+          "enum": [
+            "taginheritance"
+          ],
+          "x-ms-enum": {
+            "name": "SettingsKind",
+            "modelAsString": true
+          }
+        }
+      },
+      "required": [
+        "kind"
+      ]
+    },
+    "TagInheritanceSetting": {
+      "description": "Tag Inheritance Setting definition.",
+      "type": "object",
+      "x-ms-discriminator-value": "taginheritance",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Setting"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/TagInheritanceProperties",
+          "title": "Tag Inheritance properties"
+        }
+      }
+    },
+    "SettingsListResult": {
+      "description": "Setting list result. It contains a list of settings.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of settings.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Setting"
+          }
+        }
+      }
+    },
+    "TagInheritanceProperties": {
+      "type": "object",
+      "description": "The properties of the tag inheritance setting.",
+      "required": [
+        "preferContainerTags"
+      ],
+      "properties": {
+        "preferContainerTags": {
+          "description": "Prefer Container tags to override container tags with resource tags in case of conflicts.",
+          "type": "boolean"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "scopeSettingParameter": {
+      "name": "scope",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The scope associated with setting. This includes 'subscriptions/{subscriptionId}' for subscription scope, 'providers/Microsoft.Billing/billingAccounts/{billingAccountId}' for Billing Account scope, 'providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}' for billing profile scope.",
```

Comment:
>"description": "The scope associated with setting. This includes 'subscriptions/{subscriptionId}' for subscription scope, 'providers/Microsoft.Billing/billingAccounts/{billingAccountId}' for Billing Account scope, 'providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}' for billing profile scope.", [](http://example.com/codeflow?start=6&length=342)

Does it mean that: all the APIs defined using {scope} in Operation Path, will exist and will be available to customers for all these 3 scopes: subscriptions, billingAccounts and billingProfiles?

Reply:
Thats correct